src/app/clusters/ias-zone-client/ias-zone-client.cpp

```diff
@@ -1,567 +0,0 @@
-/**
```

Comment:
I'm a little worried about removing the cluster impls, since that will make it harder in the future to remember they ever existed and resurrect them....

Reply:
Sigh. I was hoping to remove that, but that is OK I just remove the related declarations in `silabs/types-silabs.xml`.

Reply:
Why are we keeping non-specced code that does not even compile or work?

If there is sufficient value in using old code, then I imagine we would resurect. However there is also the possibility that the work to try to adapt things to a new spec is more than just implementing the spec.

Reply:
> Why are we keeping non-specced code that does not even compile or work?

It _is_ specced in Zigbee.  The question is how that spec will be adapted for Matter.

Reply:
One thought: we remove the code but put a note in the dirs that says the code was removed pending a Matter spec.  Then people trying to find the code can easily "git log" those dirs instead of having to guess when things got removed and how to find them in history.